x/blob/client/testutil/integration_test.go

```diff
@@ -182,7 +177,9 @@ func (s *IntegrationTestSuite) TestSubmitPayForBlob() {
 	}
 }
 
-// The "_Flaky" suffix indicates that the test may fail non-deterministically especially when executed in CI.
-func TestIntegrationTestSuite_Flaky(t *testing.T) {
+func TestIntegrationTestSuite(t *testing.T) {
```

Comment:
Can't repro locally

```
$ go test ./... -count=3 -run "TestIntegrationTestSuite"
ok  	github.com/celestiaorg/celestia-app/x/blob/client/testutil	64.982s
```

Reply:
we could get this to go way faster if we refactored it to not use the cosmos-sdk's network code https://github.com/celestiaorg/celestia-app/issues/829